ui/app/js/controllers/required_signoffs_controller.js

```diff
@@ -43,6 +43,27 @@ function($scope, $modal, $q, ProductRequiredSignoffs, PermissionsRequiredSignoff
     }
   }, true);
 
+  $scope.undoRemoveRole = function(roleName, rs) {
+    var product = Object.keys(rs)[0];
+    var channel = Object.keys(rs[product].channels)[0];
+
+    $scope.required_signoffs[product].channels[channel][roleName].sc.signoffs_required = 1;
```

Comment:
Does this serve a purpose? You're setting sc to null if the http request succeeds a few lines down.

Reply:
@mozbhearsum The line was serving no purpose for sure, I have since removed it. Thanks.